editor/src/clj/editor/atlas.clj

```diff
@@ -758,12 +758,22 @@
    page     :- types/Int32
    geometry :- s/Any])
 
+(defn rotate-vertices-90-cw [vertices]
+  (mapcat (fn [[x y]]
+            [y (- x)])
+          (partition 2 vertices)))
```

Comment:
I'd prefer us not to use lazy sequences, and instead implement such transformations using transducers, i.e.:
```clj
(into []
      (comp (partition-all 2)
            (mapcat (fn [[x y]]
                      [y (- x)])))
      vertices)
```

Reply:
fixed, thank you